client/webserver/site/src/css/market.scss

```diff
@@ -417,7 +417,8 @@ div[data-handler=markets] {
 
       &:hover,
       &.selected {
-        border-color: #6cac78;
+        border-color: #dd7900;
+        color: #dd7900;
```

Comment:
I feel like we need to apply the selected attribute on page load too.  When I select a bin size, it gets these colors, but when I refresh the page, each bin size button looks the same.